osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/repositories/DlmsSecurityKeyRepository.java

```diff
@@ -17,15 +17,22 @@
 import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
+/**
+ * @deprecated with the introduction of secret-management, the security_key table has become obsolete.
+ */
+@Deprecated
 @Repository
 public interface DlmsSecurityKeyRepository extends JpaRepository<SecurityKey, Long> {
 
+    @Deprecated
     @Query("SELECT s FROM SecurityKey s JOIN FETCH s.dlmsDevice WHERE s.securityKeyType = (:securityKeyType)")
     public List<SecurityKey> findBySecurityKeyType(@Param("securityKeyType") SecurityKeyType securityKeyType);
 
+    @Deprecated
     SecurityKey findByDlmsDeviceAndSecurityKeyTypeAndValidToIsNull(DlmsDevice dlmsDevice,
             SecurityKeyType securityKeyType);
 
+    @Deprecated
     SecurityKey findByDlmsDeviceAndSecurityKeyTypeAndValidToNotNull(DlmsDevice dlmsDevice,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1133)